server/services/conditionService.test.ts

```diff
@@ -178,4 +180,55 @@ describe('ConditionService', () => {
       expect(conditionService.parseResponse(conditionConfig)).toEqual(expectedOutput)
     })
   })
+
+  describe('Get parent or self licence additional conditions', () => {
```

Comment:
Minor, but the description text should probably be updated to reference the new method name rather than the old one